File: golang/client/client_test.go
package client

import (
	"os"
	"testing"

	"github.com/alibabacloud-go/tea/tea"
	"github.com/alibabacloud-go/tea/utils"
)

func Test_Env(t *testing.T) {
	tmp := os.Getenv("TMP")
	err := SetEnv(tea.String("TMP"), tea.String("value"))
	utils.AssertNil(t, err)

	e := GetEnv(tea.String("TMP"))
	utils.AssertEqual(t, tea.StringValue(e), "value")

	err = SetEnv(tea.String("TMP"), tea.String(tmp))
	utils.AssertNil(t, err)
}

File: csharp/tests/ClientTest.cs
using System;
using Xunit;
using AlibabaCloud.DarabonbaEnv;

namespace tests
{
    public class ClientTest
    {
        [Fact]
        public void Test_Env()
        {
            Client.SetEnv("TMP", "value");
            Assert.Equal("value", Client.GetEnv("TMP"));
        }
    }
}
